src/main/scala/mesosphere/marathon/api/v2/PodsResource.scala

```diff
@@ -40,22 +40,7 @@ class PodsResource @Inject() (
   // update the json first
   private def normalize(pod: Pod): Pod = identity(pod)
   // If we can normalize using the internal model, do that instead.
-  private def normalize(pod: PodDefinition): PodDefinition = {
-    if (pod.networks.exists(_.name.isEmpty)) {
-      val networks = pod.networks.map { network =>
-        if (network.name.isEmpty) {
-          config.defaultNetworkName.get.fold(network) { name =>
-            network.copy(name = Some(name))
-          }
-        } else {
-          network
-        }
-      }
-      pod.copy(networks = networks)
-    } else {
-      pod
-    }
-  }
+  private def normalize(pod: PodDefinition): PodDefinition = identity(pod)
```

Comment:
It'll make the conversion to akka http easier. Trying to get us thinking in a pattern ;)